action.py

```diff
@@ -244,6 +295,21 @@ def check_bool(value: str) -> bool:
 def main(config: tuple[ConfigFromEnv, ConfigFromFile]):
     config_from_env, config_from_file = config
 
+    def validate_volumes():
+        volume_names = tuple(
+            volume.name
+            for volume in config_from_file.volumes
+        )
+        if len(volume_names) != len(set(volume_names)):
+            raise ValueError("Non-unique volume names")
```

Comment:
Theoretisch kann man sich das sparen, wenn man `volume.name` zu einem Key in der Config macht. In dem Fall kann es per Definition keine doppelten Namen geben. Allerdings hat man dann den Nachteil, das ein Teil des Objekts außerhalb des Objekts liegt, nämlich im Key. Daher bin ich hier +/-0.

Reply:
s.o.